flipperdevices/flipperzero-firmware
# Switch sub-1GHz band

Add RF bands control to F5. No effect in F4.

1. Go to Sub-1GHz app
2. Set any freq by pressing up and down
3. Select band by pressing left/right
4. Trigger transmission by holding OK button
5. Measure transmitting power

## How to check schematic
1. Select Band: Isolation, check that no RF power on air
2. Select Band 1. Check RF power, shorting C39 must decrease power, shorting C43, C45 must have no effect
3. Select Band 2. Check RF power, shorting C43 must decrease power, shorting C39, C45 must have no effect
4. Select Band 3. Check RF power, shorting C45 must decrease power, shorting C39, C43 must have no effect


# Checklist (do not modify)

- [x] RF Check by @skotopes
